.github/workflows/release.yml

```diff
@@ -61,6 +61,6 @@ jobs:
       # Sync docs to rdmd.readme.io
       #
       - name: Sync docs to rdmd.readme.io
-        uses: readmeio/rdme@v9
+        uses: readmeio/rdme@next
         with:
-          rdme: docs ./docs --key=${{ secrets.RDME_KEY }} --version=2
+          rdme: docs upload ./docs --key=${{ secrets.RDME_KEY }} --version=2
```

Comment:
i heard rumblings that we're consolidating these docs into the main project but i believe this API key points to the markdown project!